script/validate-data/index.ts

```diff
@@ -44,6 +44,7 @@ const propertiesSchema = {
 async function checkWorkflows(folders: string[], allowed_categories: object[]): Promise<WorkflowWithErrors[]> {
   const result: WorkflowWithErrors[] = []
   const workflow_template_names = new Set()
+  const workflow_template_paths = new Set()
```

Comment:
rename `workflow_template_paths` to `workflow_template_file_names` for better readability.